CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+## Version 2.0.2
+- Improved Syzkaller support.
+- Support narrower bisection for regression/fix ranges.
+- Improve Rust crash detection signatures.
+- Improved Android KASan support.
+- Batch datastore operations more aggressively.
+- Various bug fixes.
```

Comment:
Can add a point on grouping algorithm improvement to catch more variants of crashes involving inline frames.

Reply:
also add
"Enable entropic fuzzing strategy in libFuzzer"
"Test past crash regressions in corpus pruning task"

Reply:
Done!